<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>638</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>945</width>
    <height>638</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>945</width>
    <height>638</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(24, 24, 24);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>682</x>
      <y>-40</y>
      <width>16</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="robar_label">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>190</y>
      <width>151</width>
      <height>221</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/simple/reverso.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>420</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 81 8pt &quot;Rockwell Extra Bold&quot;;</string>
    </property>
    <property name="text">
     <string>ROBAR UNA CARTA</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>450</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>150</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>510</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 81 10pt &quot;Rockwell Extra Bold&quot;;</string>
    </property>
    <property name="text">
     <string>GRITAAA!!!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 81 12pt &quot;Rockwell Extra Bold&quot;;</string>
    </property>
    <property name="text">
     <string>TURNO DE:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>70</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 81 9pt &quot;Rockwell Extra Bold&quot;;</string>
    </property>
    <property name="text">
     <string>COLOR:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_turno">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Pepe</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_color_carta">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>70</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Rojo</string>
    </property>
   </widget>
   <widget class="QLabel" name="carta_centro">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>131</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Alito/sprites/simple/+2_rojo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="dccuatro_button">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>540</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 81 15pt &quot;Rockwell Extra Bold&quot;;
background-color: rgb(51, 51, 51);</string>
    </property>
    <property name="text">
     <string>¡¡DCCuatro!!</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>530</y>
      <width>441</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_1"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>441</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>110</y>
      <width>121</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>111</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </widget>
   <widget class="QWidget" name="widget_colores" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 81 12pt &quot;Rockwell Extra Bold&quot;;</string>
     </property>
     <property name="text">
      <string>Elija color:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="boton_verde">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 81 11pt &quot;Rockwell Extra Bold&quot;;
background-color: rgb(0, 170, 0);</string>
     </property>
     <property name="text">
      <string>VERDE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="boton_rojo">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 81 11pt &quot;Rockwell Extra Bold&quot;;
background-color: rgb(170, 0, 0);</string>
     </property>
     <property name="text">
      <string>ROJO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="boton_amarillo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 81 11pt &quot;Rockwell Extra Bold&quot;;
background-color: rgb(255, 255, 2);</string>
     </property>
     <property name="text">
      <string>AMARILLO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="boton_azul">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 81 11pt &quot;Rockwell Extra Bold&quot;;
background-color: rgb(0, 0, 127);</string>
     </property>
     <property name="text">
      <string>AZUL</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_ganaste" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 81 25pt &quot;Rockwell Extra Bold&quot;;</string>
     </property>
     <property name="text">
      <string>GANASTE REY!!</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_perdiste" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 81 12pt &quot;Rockwell Extra Bold&quot;;</string>
     </property>
     <property name="text">
      <string>PERDISTE :(</string>
     </property>
    </widget>
    <widget class="QPushButton" name="boton_seguir_viendo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 81 11pt &quot;Rockwell Extra Bold&quot;;
background-color: rgb(0, 170, 0);</string>
     </property>
     <property name="text">
      <string>SEGUIR VIENDO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="boton_irse">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 81 11pt &quot;Rockwell Extra Bold&quot;;
background-color: rgb(170, 0, 0);</string>
     </property>
     <property name="text">
      <string>IRSE</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>ingresar_usuario()</slot>
 </slots>
</ui>
